src/javascript/core/send.js

```diff
@@ -122,6 +122,9 @@ function add(request) {
 function run(callback) {
 	if (utils.isUndefined(callback)) {
 		callback = function () {};
+	} else if (typeof(callback) !== 'function') {
+		utils.log(`o-tracking error: 'callback' is defined, but is not a function.`)
```

Comment:
Thanks Adam!!
Will these be logged beyond heroku logs? Sentry?
Also, would it help to add the `callback` itself to the logs for debugging?